Adyen/Config.cs

```diff
@@ -51,9 +51,12 @@ public class Config
         public string CloudApiEndPoint { get; set; }
         [Obsolete("This is deprecated property by Adyen.")]
         public bool SkipCertValidation { get; set; } = false;
-      
-        //POS Terminal Management 
+
+        //POS Terminal Management
         public string PosTerminalManagementEndpoint { get; set; }
 
+
+        public bool HasPassword => !string.IsNullOrEmpty(Password);

```

Comment:
could we use getters for these
```suggestion
       public bool HasPassword
        {
            get { return !string.IsNullOrEmpty(Password); }
        }

```

Reply:
Well, of course, we could, but it has no sense because it's just a syntax sugar that was introduced to reduce the boilerplate